Bottom: 79295cbf1b1f5338fd10344885d0583ab3db49e4
Top:    5bd99ed0603e1be1979aa1d8886efdac63517f62
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-24 16:54:12 +0000

Refresh of add-some-shared-code-for-gl-dr

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index c95cc7f..5cba0b4 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -157,7 +157,7 @@ draw_grid ()
     {
       npoints = n + 10;
       points =
-	MyRealloc (points, npoints * 2 * sizeof (GLfloat), "gtk_draw_grid");
+        MyRealloc (points, npoints * 2 * sizeof (GLfloat), "gtk_draw_grid");
     }
 
   glEnableClientState (GL_VERTEX_ARRAY);
@@ -173,7 +173,7 @@ draw_grid ()
     {
       int vy = Vy (y);
       for (i = 0; i < n; i++)
-	points[2 * i + 1] = vy;
+        points[2 * i + 1] = vy;
       glDrawArrays (GL_POINTS, 0, n);
     }
 
@@ -190,8 +190,8 @@ int
 hidgl_set_layer (const char *name, int group, int empty)
 {
   int idx = (group >= 0
-	     && group <
-	     max_layer) ? PCB->LayerGroups.Entries[group][0] : group;
+             && group <
+             max_layer) ? PCB->LayerGroups.Entries[group][0] : group;
 
   if (idx >= 0 && idx < max_layer + 2) {
     gport->trans_lines = TRUE;
@@ -200,28 +200,28 @@ hidgl_set_layer (const char *name, int group, int empty)
   if (idx < 0)
     {
       switch (SL_TYPE (idx))
-	{
-	case SL_INVISIBLE:
-	  return /* pinout ? 0 : */ PCB->InvisibleObjectsOn;
-	case SL_MASK:
-	  if (SL_MYSIDE (idx) /*&& !pinout */ )
-	    return TEST_FLAG (SHOWMASKFLAG, PCB);
-	  return 0;
-	case SL_SILK:
+        {
+        case SL_INVISIBLE:
+          return /* pinout ? 0 : */ PCB->InvisibleObjectsOn;
+        case SL_MASK:
+          if (SL_MYSIDE (idx) /*&& !pinout */ )
+            return TEST_FLAG (SHOWMASKFLAG, PCB);
+          return 0;
+        case SL_SILK:
 //          gport->trans_lines = TRUE;
           gport->trans_lines = FALSE;
-	  if (SL_MYSIDE (idx) /*|| pinout */ )
-	    return PCB->ElementOn;
-	  return 0;
-	case SL_ASSY:
-	  return 0;
-	case SL_RATS:
-	  gport->trans_lines = TRUE;
-	  return 1;
-	case SL_PDRILL:
-	case SL_UDRILL:
-	  return 1;
-	}
+          if (SL_MYSIDE (idx) /*|| pinout */ )
+            return PCB->ElementOn;
+          return 0;
+        case SL_ASSY:
+          return 0;
+        case SL_RATS:
+          gport->trans_lines = TRUE;
+          return 1;
+        case SL_PDRILL:
+        case SL_UDRILL:
+          return 1;
+        }
     }
   return 0;
 }
@@ -450,7 +450,7 @@ hidgl_draw_arc (hidGC gc, double width, int vx, int vy,
   if (flip_y)
     {
       start_angle = - start_angle;
-      delta_angle = - delta_angle;					
+      delta_angle = - delta_angle;
     }
   /* make sure we fall in the -180 to +180 range */
   start_angle = (start_angle + 360 + 180) % 360 - 180;
